Bottom: b4a6e179a9cb6fa40afe3b8b9058bcbbf72c2a3c
Top:    0c546270335c0791b3545dc672c8a2ba89a4e413
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 15:53:04 +0100

Refresh of hid-gtk-separate-out-board-fli

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 08acb49..183a640 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1555,20 +1555,12 @@ CursorAction(int argc, char **argv, Coord x, Coord y)
     AFAIL (cursor);
 
   dx = GetValueEx (argv[1], argv[3], NULL, extra_units_x, "");
-<<<<<<< current
-  if (gport->view.flip_x)
-    dx = -dx;
-  dy = GetValueEx (argv[2], argv[3], NULL, extra_units_y, "");
-  if (!gport->view.flip_y)
-    dy = -dy;
-=======
   dy = GetValueEx (argv[2], argv[3], NULL, extra_units_y, "");
 
 #if 0 /* We cannot know this sensibly from the renderer, so we have to remove it */
-  if (ghid_flip_x) dx = -dx;
-  if (!ghid_flip_y) dy = -dy;
+  if (gport->view.flip_x) dx = -dx;
+  if (gport->view.flip_x) dy = -dy;
 #endif
->>>>>>> patched
 
   EventMoveCrosshair (Crosshair.X + dx, Crosshair.Y + dy);
   gui->set_crosshair (Crosshair.X, Crosshair.Y, pan_warp);
@@ -1949,28 +1941,6 @@ HID_Action ghid_main_action_list[] = {
 REGISTER_ACTIONS (ghid_main_action_list)
 
 
-<<<<<<< current
-static int
-flag_flipx (int x)
-{
-  return gport->view.flip_x;
-}
-
-static int
-flag_flipy (int x)
-{
-  return gport->view.flip_y;
-}
-
-HID_Flag ghid_main_flag_list[] = {
-  {"flip_x", flag_flipx, 0},
-  {"flip_y", flag_flipy, 0}
-};
-
-REGISTER_FLAGS (ghid_main_flag_list)
-
-=======
->>>>>>> patched
 #include "dolists.h"
 
 /*
